framework/web/Session.php

```diff
@@ -543,15 +552,16 @@ public function getTimeout()
     public function setTimeout($value)
     {
         $this->freeze();
-        ini_set('session.gc_maxlifetime', $value);
+        ini_set('session.gc_maxlifetime', (int) $value);
```

Comment:
ini_set expects a string.

Reply:
@My6UoT9 yes, php convert int to string

Reply:
It must be a string anyway when passed to ini_set. Cast it again.

Reply:
I'd remove the cast instead.